### PR TITLE
Added Swift Logo to the Readme

### PR DESCRIPTION
**Pull request by codestergit**
We need this !!

### CODE REVIEW

**Comment by emilstahl** [comment]
Why not SVG?
`https://upload.wikimedia.org/wikipedia/commons/archive/9/9d/20150707005015%21Swift_logo.svg`

**Comment by codestergit** [comment]
@krememek  What do you prefer svg or png. Should I keep this in local swift repo ??

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+![Swift-Logo](https://devimages.apple.com.edgekey.net/assets/elements/icons/swift/swift-128x128.png)
```
Comment (krememek) [inline README.md]: My preference would be to link to an asset from `swift.org`.  The primary SVG should used as a the website header would probably work.

File: README.md [hunk 2]
```diff
@@ -141,4 +142,4 @@ See docs/Testing.rst.
 
 ## Contributing to Swift
 
-Contributions to Swift are welcomed and encouraged! Please see the [Contributing to Swift guide](https://swift.org/contributing/).
```
Comment (krememek) [inline README.md]: Also, was there any change here?  Was it just a whitespace change?  I can't tell in the diff.

File: README.md (hunk 1 above)
Comment (emilstahl) [inline README.md]: @krememek Of course! Done in #113

**Comment by codestergit** [comment]
@krememek Done the changes. Thanks

File: README.md (hunk 2 above)
Comment (codestergit) [inline README.md]: I have not changed here. I do not know why it I got conflict. Done the changes. Thanks

**Comment by emilstahl** [comment]
@codestergit Great! However the logo is quite big. I just fixed the width in my pull request #113.

**Comment by codestergit** [comment]
@krememek  Reduced height please verify

**Comment by zachpanz88** [comment]
As the code you suggested uses html, you could use the following code which is styled with markdown:

```
![Swift Logo](https://swift.org/assets/images/swift.svg =x70)
```
Which results in:
Swift Logo

**Comment by emilstahl** [comment]
@zachpanz88 It does not work.

**Comment by zachpanz88** [comment]
Suggestion removed as I believe GitHub does not support SVG images in markdown, but html will work.

**Comment by KamranMackey** [comment]
@emilstahl Yes it does, I just previewed it in the GitHub markdown preview. It does work.

**Comment by emilstahl** [comment]
@zachpanz88 Did you see the code @zachpanz88 posted and then deleted? It clearly did not work.

**Comment by zachpanz88** [comment]
This code will work, but changing the size they way I originally did will not work, which was the error in the first version.
```
![Swift Logo](https://swift.org/assets/images/swift.svg)
```

Output:
Swift Logo

**Comment by iosdv** [comment]
@zachpanz88  You are right there is some problem with github markup we can not give the height. I have previously faced that issue. You need to add HTML code for that. Thanks

**Comment by iosdv** [comment]
Looks good to me :+1: +

**Comment by krememek** [comment]
Looks great.

**Pull request merged**